Panels/Header.js

```diff
@@ -390,6 +391,10 @@ const HeaderBase = kind({
 		}
 	},
 
+	handlers: {
+		onBack: forwardCustom('onBack')
```

Comment:
Let's add `onClose` handler also to be used for a close button.

Reply:
🆗